migrations/schema-15.sql

```diff
@@ -1003,3 +1009,46 @@ CREATE EVENT TRIGGER pgrst_drop_watch ON sql_drop
 -- Dbmate schema migrations
 --
 
+INSERT INTO public.schema_migrations (version) VALUES
+    ('00000000000000'),
+    ('00000000000001'),
+    ('00000000000002'),
+    ('00000000000003'),
+    ('10000000000000'),
+    ('20211115181400'),
+    ('20211118015519'),
+    ('20211122051245'),
+    ('20211124212715'),
+    ('20211130151719'),
+    ('20220118070449'),
+    ('20220126121436'),
```

Comment:
Q: Why is this needed? What service keeps track of these migration rows?

Also, it's duplicated across several files. Is there a way to DRY this?

Reply:
hmm these were actually generated from `nix run .#dbmate-tool -- --version all`

Reply:
Right, this was clarified on https://github.com/supabase/postgres/pull/1555#discussion_r2049738715.